packages/@aws-cdk/toolkit-lib/lib/api/io/private/codes.ts

```diff
@@ -125,18 +209,31 @@ export const CODES = {
     description: 'Provides destroy times',
     interface: 'Duration',
   }),
-  CDK_TOOLKIT_I7010: make.info({
+  CDK_TOOLKIT_I7010: make.confirm<ConfirmationRequest>({
     code: 'CDK_TOOLKIT_I7010',
     description: 'Confirm destroy stacks',
+    interface: 'ConfirmationRequest',
+  }),
+  CDK_TOOLKIT_I7100: make.info<StackDestroyProgress>({
+    code: 'CDK_TOOLKIT_I7100',
+    description: 'Stack destroy progress',
+    interface: 'StackDestroyProgress',
+  }),
+
+  CDK_TOOLKIT_I7900: make.result<cxapi.CloudFormationStackArtifact>({
+    code: 'CDK_TOOLKIT_I7900',
+    description: 'Stack deletion succeeded',
+    interface: 'cxapi.CloudFormationStackArtifact',
```

Comment:
already discussed but note that this will result in a bad link in the CODE_REGISTRY

Reply:
Acceptable right now. Will fix later.